tests/custom_lookups/tests.py

```diff
@@ -63,6 +63,15 @@ def as_sql(self, compiler, connection):
         return '3 * (%s)' % lhs, lhs_params
 
 
+class LastDigitBilateralTransform(models.Transform):
+    bilateral = True
```

Comment:
Does this make a difference to the tests? I'm not a fan of the bilateral attribute - it was introduced before expressions were a thing. Not sure we want to propagate their use.

Reply:
Removed `bilateral` from this transform.